marc_to_solr/spec/lib/princeton_marc_spec.rb

```diff
@@ -583,6 +532,10 @@
       expect(@subjects).not_to include("split genre")
     end
 
+    it 'excludes non-approved subfield $2 vocab types' do
+      expect(@subjects).not_to include("Bad subject")
+    end
+
```

Comment:
Can you add some test coverage verifying that the Siku subjects show up?

Reply:
@escowles the siku subjects have been displaying in staging for a few months now: https://catalog-staging.princeton.edu/catalog/1204912

Reply:
Through this PR https://github.com/pulibrary/marc_liberation/pull/549